### ISSUE TITLE
Add "skip to main content" link

### ISSUE THREAD

**Issue by mark-wiemer** [mark-wiemer/hello-hello#34]
Should be hidden unless user keyboard-navigates to it. Should take user to the `main` section. Main heading should be in `main` section, so update `Layout` as well.

**Comment by mark-wiemer** [comment]
Closing as fixed with #45
